crates/vm/levm/src/vm.rs

```diff
@@ -709,14 +709,13 @@ impl VM {
             .and_then(|gas| gas.checked_add(refunded_gas))
             .ok_or(VMError::Internal(InternalError::UndefinedState(0)))?;
 
-        let gas_return_amount = self
+        let gwei_return_amount = self
             .env
             .gas_price
-            .low_u64()
-            .checked_mul(gas_to_return)
+            .checked_mul(U256::from(gas_to_return))
             .ok_or(VMError::Internal(InternalError::UndefinedState(1)))?;
 
-        self.increase_account_balance(sender_address, U256::from(gas_return_amount))?;
+        self.increase_account_balance(sender_address, gwei_return_amount)?;
```

Comment:
Ditto

Reply:
Done in [ee33452](https://github.com/lambdaclass/ethrex/pull/1464/commits/ee33452a232dc510dd4332edb5df43fdf44dfc92)